airbyte-integrations/connectors/source-slack/source_slack/schemas/channel_messages.json

```diff
@@ -184,8 +184,10 @@
       "type": ["null", "string"]
     },
     "ts": {
-      "type": ["null", "string"],
-      "format": "date-time"
+      "type": ["null", "string"]
+    },
+    "id": {
```

Comment:
see comment below, we should remove the `id` field since it is redundant with `ts`

Reply:
Done